### PR TITLE
Update utils.py

### PR DESCRIPTION
**Pull request by CodeLongAndProsper90**
This adds support for `Feh` and `Nitrogen`, if Ubuntu or Elementary is not found.
This has no effect on existing options, and only comes into play if gsettings is not found.

### CODE REVIEW

File: backgroundchanger/utils.py [hunk 1]
```diff
@@ -30,9 +31,9 @@ def get_keys():
         data = json.loads(key_file.read())
     if 'secret_key' not in data:
         return {
-            'access_key': data['access_key'],
-            'secret_key': None
-        }
+                'access_key': data['access_key'],
```
Comment (conruggles) [inline backgroundchanger/utils.py:34]: Please remove these spacing changes

File: backgroundchanger/utils.py [hunk 2]
```diff
@@ -52,9 +53,9 @@ def get_screen_size():
     """
     root = Tk()
     return {
-        'height': root.winfo_vrootheight(),
-        'width': root.winfo_vrootwidth()
-    }
+            'height': root.winfo_vrootheight(),
```
Comment (conruggles) [inline backgroundchanger/utils.py:56]: Please remove these spacing changes

File: backgroundchanger/utils.py [hunk 3]
```diff
@@ -66,24 +67,41 @@ def get_background_cmd(photo_name: str):
...
-                'org.gnome.desktop.background',
-                'picture-uri',
-                'file://' + photo_name
-            ]
+                    'gsettings',
+                    'set',
+                    'org.gnome.desktop.background',
+                    'picture-uri',
+                    'file://' + photo_name
+                    ]
```
Comment (conruggles) [inline backgroundchanger/utils.py:83]: please revert the spacing on this return statement

File: backgroundchanger/utils.py [hunk 4]
```diff
@@ -66,24 +67,41 @@ def get_background_cmd(photo_name: str):
...
+                        photo_name
+                        ]
+
+
     elif system == 'Windows':
         raise ValueError(
-            'Windows is not yet implemented to change the background. However, you can still change the background. photo name: {}'.format(
-                photo_name))
+                'Windows is not yet implemented to change the background. However, you can still change the background. photo name: {}'.format(
+                    phot o_name))
```
Comment (conruggles) [inline backgroundchanger/utils.py]: fix variable name
```suggestion
                    photo_name))
```

File: backgroundchanger/utils.py [hunk 5]
```diff
@@ -66,24 +67,41 @@ def get_background_cmd(photo_name: str):
...
     raise ValueError(
-        '{} is not yet implemented to change the background. However, you can still change the background. photo name: {}'.format(
-            system, photo_name))
+            '{} is not ye t implemented to change the background. However, you can still change the background. photo name: {}'.format(
```
Comment (conruggles) [inline backgroundchanger/utils.py]: fix word typo
```suggestion
            '{} is not yet implemented to change the background. However, you can still change the background. photo name: {}'.format(
```

File: README.md [hunk 6]
```diff
@@ -12,6 +12,8 @@ This is pretty light on dependencies, but you will need to make sure you have a
 * `requests` - python module for HTTP requests.
 * `pywal` - generates color schemes and used for changing the wallpaper.
 
+This optionally requres `feh` or `nitrogen` if you are not using Ubuntu or Rlementary OS.
```
Comment (conruggles) [inline README.md]: typo
```suggestion
This optionally requres `feh` or `nitrogen` if you are not using Ubuntu or Elementary OS.
```

**Review by conruggles — CHANGES_REQUESTED** [review]
Please address the comments I left

**Comment by CodeLongAndProsper90** [comment]
Ah yes, sorry about those changes, my IDE was a little overzealous.

**Review by conruggles — CHANGES_REQUESTED** [review]
Still some spacing changes it looks like, on the return statements

**Comment by CodeLongAndProsper90** [comment]
I think I'll close this one and open a new pr, using a different ide. Or just fix Vim.

**Pull request closed without merging**